powerhome/playbook
# [PLAY-795] Converting Walkthrough kit to Typescript

**What does this PR do?**
Converting Walkthrough kit to Typescript.

[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/PLAY-795)

**How to test?**
1. Go to the Walkthrough kit on Playbook docs
2. Ensure all the examples still work

#### Checklist:
